src/components/NavBar/NavBar.js

```diff
@@ -21,17 +24,26 @@ const NavBar = () => {
                         alt="acm logo"
                         onClick={() => navigate("/")} />
                 </div>
-                <div className="element-container">
+                
+                <div className="element-container resouce-btn">
                     <DropDownButton
                         text="Resources"
                         isExpanded={isMenuExpanded}
                         onClick={() => setMenuExpanded(prev => !prev)}
                     />
                 </div>
+                
+                <button className={isMenuExpanded ? 'active' : ''} id="hamburger" type="button" onClick={() => setMenuExpanded(prev => !prev)} aria-label="navigation menu" aria-expanded={isMenuExpanded} tabIndex="0">
+                    <span className= "bar" id="bar-one"></span>
+                    <span className= "bar" id="bar-two"></span>
+                    <span className="bar" id="bar-three"></span>
+                </button>
             </div>
-            {isMenuExpanded ? <DropDownMenu resources={resources}/> : null}
+            {isMenuExpanded ? <DropDownMenu resources={resources} /> : null}
+
+
```

Comment:
you can do this!
```suggestion
            {isMenuExpanded && <DropDownMenu resources={resources} />}
```

Reply:
Interesting. I didn't about short circuit evaluations.